### ISSUE TITLE
error loading and saving one excel file with broken references

### ISSUE THREAD

**Issue by RogerioDuarte** [ClosedXML/ClosedXML#1299]
**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML** 
0.94.0.0

**What is the current behavior?**
 
save command fails when one existing xlsx file has broken references

**What is the expected behavior or new feature?**

save command to work

**Did this work in previous versions of our tool?  Which versions?**

not tested

## Reproducibility

**Code to reproduce problem:**
c#
```
public void myTest_Load_and_Save(string filePath)
{
 var workbook_from = new XLWorkbook(filePath);
 workbook_from.Save();
}
```
the error is this:
```
System.NullReferenceException: 'Object reference not set to an instance of an object.'

v was null.

--------------------------------------------------------
var yyValues = yy.Values.Values.Where(v => !v.IsFormula).Select(v => v.Value);
--------------------------------------------------------


file : 
C:\ClosedXML-develop\ClosedXML-develop\ClosedXML\Excel\ConditionalFormats\XLConditionalFormat.cs

line code number : 32
--------------------------------------------------------
```

[Stamped_Formulas.xlsx](https://github.com/ClosedXML/ClosedXML/files/3771772/Stamped_Formulas.xlsx)

- [x] I attached a sample spreadsheet.  (You can drag files on to this issue)

**Comment by RogerioDuarte** [comment]
this can be a solution:

 var xxValues = xx.Values.Values.Where(v => (v != null) && (!v.IsFormula)).Select(v => v.Value);
                var yyValues = yy.Values.Values.Where(  v => (v != null ) && (!v.IsFormula) ).Select(v => v.Value );
                var xxFormulas = x.Ranges.Any() ? xx.Values.Values.Where(v => (v != null) && (v.IsFormula)).Select(f => ((XLCell)x.Ranges.First().FirstCell()).GetFormulaR1C1(f.Value)) : null;
                var yyFormulas = y.Ranges.Any() ? yy.Values.Values.Where(v => (v != null) && (v.IsFormula)).Select(f => ((XLCell)y.Ranges.First().FirstCell()).GetFormulaR1C1(f.Value)) : null;

**Comment by Pankraty** [comment]
The issue does not reproduce on 0.94 and later.

It existed in 0.93.1 but has been fixed in #987
